modules/sdk/gradle-plugins-source-formatter/README.markdown

```diff
@@ -1,40 +1,36 @@
 # Source Formatter Gradle Plugin
 
-The plugin allows to format the project files using the Liferay Source Formatter
-tool.
+The Source Formatter Gradle plugin allows you to format project files using the
+Liferay Source Formatter tool.
 
 ## Usage
 
 To use the plugin, include it in your build script:
 
-```gradle
```

Comment:
I removed it because we've never used this syntax before in our Markdown (mostly because our Markdown converter for LDN doesn't handle this). I do think it's helpful, and since it's viewable on Github, we could keep it in. @jhinkey what do you think?